test/solc/CommandLineParser.cpp

```diff
@@ -422,6 +420,60 @@ BOOST_AUTO_TEST_CASE(standard_json_mode_options)
 	BOOST_TEST(parsedOptions.value() == expectedOptions);
 }
 
+BOOST_AUTO_TEST_CASE(invalid_combinations)
```

Comment:
We'll probably generalize it later but for now this tests only really deals with this single option:
```suggestion
BOOST_AUTO_TEST_CASE(experimental_via_ir_invalid_input_modes)
```

Reply:
This is fixed